spec/conformance.html

```diff
@@ -1,7 +1,7 @@
 <emu-clause id="conformance">
   <h1>Conformance</h1>
   <p>
-    A conforming implementation of the ECMAScript 2023 Internationalization API Specification must conform to the ECMAScript 2023 Language Specification (ECMA-262 14<sup>th</sup> Edition, or successor), and must provide and support all the objects, properties, functions, and program semantics described in this specification.
+    A conforming implementation of the ECMAScript 2023 Internationalization API Specification must conform to the ECMAScript 2023 Language Specification (ECMA-262 14<sup>th</sup> Edition, or successor), and must provide and support all the objects, properties, functions, and program semantics described in this specification. Nothing permitted by this specification is prohibited by ECMA-262 to an implementation that does not conform with this specification.
```

Comment:
this wording is a bit confusing. can you help me understand what it's trying to say?

Reply:
That ECMA-402 does not grant powers otherwise unavailable to an ECMA-262 implementation. An example violation is https://github.com/tc39/proposal-temporal/issues/2192 , which observes that an ECMA-262 implementation will be _required_ to throw on `Temporal.Calendar.from("gregory")` unless it also implements ECMA-402.